modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/RapidExperimentationTest.java

```diff
@@ -0,0 +1,328 @@
+
+import com.intuit.wasabi.tests.library.TestBase;
+import com.intuit.wasabi.tests.library.util.Constants;
+import com.intuit.wasabi.tests.model.Application;
+import com.intuit.wasabi.tests.model.Assignment;
+import com.intuit.wasabi.tests.model.Bucket;
+import com.intuit.wasabi.tests.model.Experiment;
+import com.intuit.wasabi.tests.model.Page;
+import com.intuit.wasabi.tests.model.User;
+import com.intuit.wasabi.tests.model.factory.AssignmentFactory;
+import com.intuit.wasabi.tests.model.factory.BucketFactory;
+import com.intuit.wasabi.tests.model.factory.ExperimentFactory;
+
+/*******************************************************************************
+ * Copyright 2017 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *******************************************************************************/
+public class RapidExperimentationTest extends TestBase {
+    /**
+     * This test class contains functional tests that check the rapid experimentation functionality
+     * 
+     * Created on 05/23/2017
+     */
+    private static final int RAPID_EXP_MAX_USERS = 5; // max cap on number of users
+    private static final int NUMBER_OF_EXPERIMENTS_PER_BATCH = 5;
+    private static final int NUMBER_OF_RAPID_EXPERIMENTS_IN_BATCH = 2; // number of rapid experiments in a batch that
+                                                                       // has combination of normal and rapid
+                                                                       // experiments
+
+    private static final String BATCH_PAGE = "rapidexperiment_page";
+    private static final String PARTIAL_BATCH_PAGE = "rapidexperiment_partialpage";
+    Experiment rapidExperiment1 = null;
+    Experiment rapidExperiment2 = null;
+    Experiment rapidExperiment3 = null;
+    Page page = null;
+    List<Experiment> expList = new ArrayList<Experiment>();
+    List<Experiment> batchExperiments = new ArrayList<Experiment>();
+    List<Experiment> experimentsListOfRapidApplication = new ArrayList<Experiment>();
+
+    Application application = null;
+
+    @BeforeClass
+    public void setUp() {
+
+        application = new Application("rapidExperimentApplication");
+
+        experimentsListOfRapidApplication = getApplicationExperiments(application);
+        pauseTerminateAndDeleteExperiments(experimentsListOfRapidApplication);
+
+        rapidExperiment1 = ExperimentFactory.createExperiment("_1" + UUID.randomUUID().toString(), -1)
```

Comment:
@abhishekjain88 i meant _1 only but anyhow @abhishekjain88 1_ is also good suggestion :)